torch/csrc/jit/codegen/cuda/test/test_gpu_tensorcore.cpp

```diff
@@ -1894,7 +2409,164 @@ TEST_F(NVFuserTest, FusionTuringMatmulNT_CUDA) {
       MmaBuilder(MmaOptions::MacroType::Turing_16_8_16, gemm_tile)
           .layout(MmaOptions::MmaInputLayout::NT);
 
-  scheduleMatmul(tv2, tv0, tv1, mma_builder, gemm_tile);
+  MatmulParam params(mma_builder);
+  params.tile_sizes = gemm_tile;
+  scheduleMatmul(tv2, tv0, tv1, params);
+
+  auto options = at::TensorOptions().dtype(at::kHalf).device(at::kCUDA, 0);
+  auto t0 = at::randn({K, M}, options);
+  auto t1 = at::randn({K, N}, options);
+
+  FusionExecutor fe;
+  NVFUSER_TEST_CUDA_ARCH_COMPILE_CHECK(
+      7, 5, fe.compileFusion(&fusion, {t0, t1}));
+
+  auto cg_outputs = fe.runFusion({t0, t1});
+
+  auto tref = t0.t().to(at::kFloat).matmul(t1.to(at::kFloat));
+
+  TORCH_CHECK(cg_outputs[0].allclose(tref, 0.0001, 0.0001));
+}
+
+// Matmul test on Turing
+TEST_F(NVFuserTest, FusionTuringMatmulTNRegDoubleBuffer_CUDA) {
```

Comment:
Is double buffering on by default? Don't see it turned on here.

Reply:
No it's not. There as a option in matmul schedule params that controls it. This was git rendering issue before rebasing.